code/datums/abilities/flock/flockmind.dm

```diff
@@ -477,10 +477,30 @@
 /datum/targetable/flockmindAbility/droneControl
 	cooldown = 0
 	icon = null
-	var/task_type
 	var/mob/living/critter/flock/drone/drone = null
 
 /datum/targetable/flockmindAbility/droneControl/cast(atom/target)
+	var/datum/component/flock_ping/selected/ping = drone.GetComponent(/datum/component/flock_ping/selected)
+	ping.RemoveComponent()
+	qdel(ping)
+	if (target == src.drone)
+		return
+	//by default we just walk somewhere
+	var/task_type = /datum/aiTask/sequence/goalbased/flock/build/targetable
+	if (flockValidEnemy(target))
+		if (ismob(target) && is_incapacitated(target))
```

Comment:
Obj critters can be caged I think

Reply:
Correct, but you can't check if they're incapacitated, so we have to choose one task to switch to. This way it seems like they shoot at it for a bit and then try to cage it, which seems fine?

Reply:
Okay